components/mpas-ocean/cime_config/buildlib

```diff
@@ -51,7 +51,7 @@ system($drvcpy) == 0 or die "ERROR: $component buildlib $drvcpy failed: $?\n";
 
 my $root_path = "$OBJROOT/$component_dir/source";
 
-my $bldcmd = "cd $root_path; ${GMAKE} all ${MAKE_ARGS} CORE=$core ESM=$esm DRIVER=$component_driver GEN_F90=true";
+my $bldcmd = "cd $root_path; ${GMAKE} all ${MAKE_ARGS} CORE=$core ESM=$esm DRIVER=$component_driver GEN_F90=false";
```

Comment:
Changing `GEN_F90=false` caused the *.inc to not be placed in
`bld/ocn/source/core_ocean/inc`
Turning it back to `GEN_F90=true`.